nin/dasBoot/music.js

```diff
@@ -15,8 +15,11 @@ function loadMusic() {
   _gainNode.connect(webAudioContext.destination);
   var _analyserNode = webAudioContext.createAnalyser();
   _analyserNode.fftSize = 2048;
+  _analyserNode.minDecibels = -90;
```

Comment:
What is the purpose of setting minDecibels to -90 instead of the default, which is -100?